app/_how-tos/kic-get-started-services-routes.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Services and Routes
+description: |
+  Use the a HTTPRoute or Ingress to configure a Service and a Route.
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/get-started/services-and-routes/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Get Started
+
+series:
+  id: kic-get-started
+  position: 2
+
+tldr:
+  q: How do I create a Service and a Route with {{ site.kic_product_name }}
+  a: |
+    Create a HTTPRoute that points to a Kubernetes service in your cluster.
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+prereqs:
+  skip_product: true
+---
+
+## How Kubernetes resources map to Kong Entities
+
+A Service inside Kubernetes is a way to abstract an application that is running on a set of Pods. This maps to two objects in Kong: Service and Upstream.
```

Comment:
```suggestion
A Service inside Kubernetes is a way to abstract an application that is running on a set of Pods. This maps to two entities in {{site.base_gateway}}: Service and Upstream.
```
For consistency, let's use "entities" instead of "objects".